packages/docs/operations-manual/key-management.md

```diff
@@ -4,65 +4,209 @@
 {% endhint %}
 
-## Key Security
+## Undesignated
+
+By default, any private key generated in the Celo protocol does not have a specific role in proof-of-stake. These keys can be used to sign and send transactions that are unrelated to Celo's proof-of-stake protocol, including but by no means limited to transferring Celo Gold, transferring Celo Dollars, exchanging Celo Gold for Celo Dollars, etc.
+
+Undesignated keys can be designated as Locked Gold Accounts or authorized as signer keys on behalf of a Locked Gold Account by sending special transactions using `celocli`.
+
+Note that [ReleaseGold](../celo-codebase/protocol/release-gold) beneficiary keys are by default Undesignated with respect to proof-of-stake, and that the `ReleaseGold` contract address is what ultimately gets designated as a Locked Gold Account.
+
+
+## Locked Gold Accounts
+
+Locked Gold Account keys have the highest level of privileges in the Celo protocol. These keys can be used to lock and unlock cGLD in order to be used in proof-of-stake. Furthermore, Locked Gold Account keys can be used to authorize other keys to sign transactions and messages on behalf of the Locked Gold Account. In *most* cases, the Locked Gold Account key has all the privileges as any authorized keys. For example, if a voter signing key is authorized, a user can place votes on behalf of the Locked Gold Account with both the authorized vote signing key *and* the Locked Gold Account key.
+
+Because of the significant priveleges afforded to the Locked Gold Account key, it is best to store this key securely and access it as infrequently as is possible. Authorizing other signer keys is one way to minimize how frequently you need to access your Locked Gold Account key.
+
+### Creating a Locked Gold Account
+An undesignated key may be designated as a Locked Gold Account key by running the following command:
+
+```shell
+# Designate the key as a Locked Gold Account
+celocli account:create --from $ADDRESS_TO_DESIGNATE --useLedger
+
+# Confirm the key was designated as a Locked Gold Account
+celocli account:show $ADDRESS_TO_DESIGNATE
+```
+
+## Authorized Vote Signers
+
+Any Locked Gold Account key may optionally authorize an undesignated key as a vote signing key. Authorized vote signing keys can vote in validator elections and on-chain governance on behalf of the Locked Gold Account.
+
+Note that the vote signing key must first be used to generate a "proof-of-possession" indicating that key's willingness to be authorized as a signer on behalf of the Locked Gold Account.
+
+
+### Authorizing a Vote Signer
+An undesignated key may be authorized as a vote signer on behalf of a Locked Gold Account by runnign the following commands:
+
+```shell
+# Create a proof-of-possession. Note that the signer private key must be available.
+celocli account:proof-of-possession --account $LOCKED_GOLD_ACCOUNT --signer $KEY_TO_AUTHORIZE --useLedger
```

Comment:
Should we actually specify `--useLedger` on every command? It seems somewhat orthogonal

Reply:
I put it here to reinforce which keys you might consider keeping on a ledger, which is part of what these docs are intended to clarify